pay-api/src/pay_api/services/base_payment_system.py

```diff
@@ -321,7 +321,7 @@ def _send_credit_notification(payment_account: PaymentAccountModel, refund_amoun
             if (user := member.get("user")) and (contacts := user.get("contacts")):
                 receiver_recipients.append(contacts[0].get("email"))
 
-        subject = f"{refund_amount} credit was added to your account "
+        subject = f"{refund_amount}$ {'credits' if refund_amount > 1 else 'credit'} was added to your account "
```

Comment:
$ should be in front of {refund_amount} ?

Reply:
haha yup